### PR TITLE
oak.md documentation update

### PR DESCRIPTION
**Pull request by joannakennedyharvard**
Added list of summary statistics to the bottom of oak.md

### CODE REVIEW

File: docs/source/oak.md [hunk 1]
```diff
@@ -51,4 +51,17 @@ delta = 20
...
 min_t = 3
-```
\ No newline at end of file
+```
+
+## List of summary statistics
+
+The outputs of the accelerometer module contains gait summary statistics for each specified participant in daily (_gait_daily.csv) or hourly windows (_gait_hourly.csv).
+
+The following variables are created in a csv file for each survey.
```
Comment (clementzach) [inline docs/source/oak.md]: I believe this should say  "for each participant" instead of "for each survey"

**Review by clementzach — APPROVED** [review]
I think this looks good after my small survey/participant change

**Comment by hackdna** [comment]
Zach, if this looks good, please resolve the conversation above, update this branch with the new changes from the base branch, then squash and merge.

**Review by clementzach — APPROVED** [review]

**Review by clementzach — APPROVED** [review]

**Comment by clementzach** [comment]
> Zach, if this looks good, please resolve the conversation above, update this branch with the new changes from the base branch, then squash and merge.

Thanks for the reminder, I didn't realize that I needed to do that

**Pull request merged**